symbols/overlay29.yml

```diff
@@ -3577,6 +3661,33 @@ overlay29:
         r0: pointer to entity
         r1: flag for whether to check for the held item
         return: bool
+    - name: TryInflictCrossEyedStatus
+      address:
+        NA: 0x2317844
+      description: |-
+        Inflicts the Cross-Eyed status condition on a target monster if possible.
+        
+        r0: user entity pointer
+        r1: target entity pointer
+        r2: flag to only perform the check for inflicting without actually inflicting
+        return: Whether or not the status could be inflicted
+    - name: TryInflictEyedropStatus
+      address:
+        NA: 0x231795C
+      description: |-
+        Inflicts the Eyedrop status condition on a target monster if possible.
+        
+        r0: user entity pointer
+        r1: target entity pointer
```

Comment:
Hmm it feels like this should return a success flag like the others. Does it leave a success flag in `r0` when it returns? Sometimes Ghidra doesn't catch it if callers never use the return value, but I like to include it in the signature if it's there.

Reply:
After double checking, I am pretty confident it does not return anything. It immediately returns after calling UpdateStatusIconFlags. It's probably this way because the function is exclusively called in ApplyItemEffect. The other functions are used for thinks like exclusive items or have their error message handled by ApplyItemEffect. This function has the error message inside.

Reply:
Okay, just wanted to make sure. Leave as is then.